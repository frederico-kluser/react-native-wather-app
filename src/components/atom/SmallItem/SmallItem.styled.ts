import styled from 'styled-components/native';

const Container = styled.View`
  margin-right: 50px;
`;

export const Hour = styled.Text`
  font-size: 13px;
`;

export const WeatherIcon = styled.Image`
  height: 32px;
  width: 32px;
  top: 13px;
`;

export const Temperature = styled.Text`
  font-size: 25px;
  top: 23px;
`;

export default Container;
