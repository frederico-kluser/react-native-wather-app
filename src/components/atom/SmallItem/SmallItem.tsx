import React from 'react';
import {weatherIcons} from '../../../styles/icons';
import Container, {Hour, Temperature, WeatherIcon} from './SmallItem.styled';

const SmallItem = ({}: any) => (
  <Container>
    <Hour>10 AM</Hour>
    <WeatherIcon source={weatherIcons.rain} />
    <Temperature>19Â°</Temperature>
  </Container>
);

export default SmallItem;
