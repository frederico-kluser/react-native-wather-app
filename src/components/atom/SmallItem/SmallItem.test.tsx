import React from 'react';
import renderer from 'react-test-renderer';
import SmallItem from './SmallItem';

test('renders correctly', () => {
  const tree = renderer.create(<SmallItem />).toJSON();
  expect(tree).toMatchSnapshot();
});
