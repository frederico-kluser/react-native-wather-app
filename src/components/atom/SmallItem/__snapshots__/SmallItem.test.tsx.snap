// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "marginRight": 50,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 13,
        },
      ]
    }
  >
    10 AM
  </Text>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/rain.png",
      }
    }
    style={
      Array [
        Object {
          "height": 32,
          "top": 13,
          "width": 32,
        },
      ]
    }
  />
  <Text
    style={
      Array [
        Object {
          "fontSize": 25,
          "top": 23,
        },
      ]
    }
  >
    19Â°
  </Text>
</View>
`;
